### ISSUE TITLE
NullReferenceException on calculating a copied worksheet

### ISSUE THREAD

**Issue by Aurelianus** [EPPlusSoftware/EPPlus#1228]
If you copy a worksheet from one package to another and execute calculate the _values field of ExcelWorksheet is sometimes null.

test code is as follows

```
[TestMethod]
public void Issue()
{
	using (var pck = new ExcelPackage())
	{
		using (var pckTemplate = new ExcelPackage("....\\MyIssue.xlsx"))
		{
			pck.Workbook.Worksheets.Add("Foo", pckTemplate.Workbook.Worksheets[1]);
		}

		pck.Workbook.Calculate(x => x.AllowCircularReferences = true);
	}
}
```
MyIssue.xlsx will be send to you via support ticket on your page.

This test runs with EP6 and not with EP7.

**Comment by JanKallman** [comment]
This is a bug. A reference to the copied worksheet worksheet remains on shared formulas causing a null reference when calculating.
I will provide a fix shortly.

**Comment by JanKallman** [comment]
Fixed in EPPlus 7.0.6
